src/datawave_cli/__init__.py

```diff
@@ -0,0 +1,6 @@
+import os
+import sys
+from pathlib import Path
+
+if os.getenv('DWV_ENV') == 'development':
+    sys.path.insert(0, Path(__file__).resolve().parents[1])
```

Comment:
why is this required? what does it get us?

Reply:
If you import anything from `datawave_cli` it runs the `__init__.py`. So what this does is if you have that development environment set it ensures that the `src` directory is on the python path. Meaning that python will check `src/` for any imports first before checking `site-packages`. This means that if you're working on something and change it locally you don't have to rebuild and install (or uninstall) the installed package to get it to run your local changes. 

I'm assuming this is what was causing the problems you were having with the `.base_interactions` import. You had a version installed that didn't have `base_interactions` cause you hadn't built it yet. Therefore when you tried `import datawave_cli.base_interactions` it was checking that install (located in `site-packages`) and didn't see a `base_interactions` file and errored out. So you had to use the relative import of `.base_interactions` which says, "whatever folder this file is located in, find a file named `base_interactions` in this same level".

Best practices in python, for clarity sake, is to avoid relative import with the exception of two cases. One is if it is deeply nested and you have to repeatedly navigate through like 5 sub modules for the import. The second is if it is tightly coupled code that is not expected to ever change relative location to the file, which I suppose your use does fit. I just don't like relative imports.

After typing out this response I realize that if you're importing a file and not running that file directly `__file__` is gonna end up pointing to the site-packages version of the file anyway so it's probably moot and not needed. But the idea was solid!

Reply:
the addition to the toml telling it where the src is thought fixed it. Although IDK if I did that here also or just in webapp.

Reply:
The toml telling it where the src is is only for running pytest though, no?